web/views.py

```
@@ -40,6 +40,9 @@
     HttpResponse,
     HttpResponseForbidden,
     JsonResponse,
+    HttpResponseNotFound,
+    HttpResponseServerError,
+    HttpRequest,
```

Ruff (0.8.2)

`django.http.HttpRequest` imported but unused

Remove unused import: `django.http.HttpRequest`

(F401)